AssignCardToSender/index.ts

```diff
@@ -0,0 +1,13 @@
+import { OctokitAction } from '../lib/OctokitAction';
```

Comment:
It is intentional that these files are not named `index.ts`, because that would be difficult to work with in dev UI
![image](https://user-images.githubusercontent.com/57188685/220303012-b2dee4e2-4e46-468e-b676-bf7f5d08c6d5.png)

vs. `index.js`, `index.js`, `index.js` - you don't know what you're editing and where to go next

Reply:
Yeah, I thought about this afterwards, indeed I had a similar discussion a few years back. I wouldn't mind keeping the name of the action as the name of file, even if it means repetition.
I will roll this back.